staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -107,7 +107,7 @@ type JobSpec struct {
 	// and other jobs to not function correctly.  However, You may see
 	// `manualSelector=true` in jobs that were created with the old `extensions/v1beta1`
 	// API.
-	// More info: https://git.k8s.io/community/contributors/design-proposals/selector-generation.md
+	// More info: https://git.k8s.io/community/blob/master/contributors/design-proposals/apps/selector-generation.md
```

Comment:
Did you try to visit this link? It doesn't work. The `blob/master/` is redundant here.

Reply:
Please modify other links below as well :)